codesandbox/sandpack
# feat(codemirror): upgrade

Todo: the new syntax-highlighting parse just works inside Storybook ([preview](https://18ycun-6006.preview.csb.app/?path=/story/presets-codemirror--jsx)), but it doesn't when running it on another application. It seems that all the parse rules are messed up when consuming it as a package.

**How to test:**
1. Checkout project or https://codesandbox.io/p/github/codesandbox/sandpack/update/codemirror
2. Open Storybook to make sure it works: `yarn dev:react` or `https://18ycun-6006.preview.csb.app/?path=/story/presets-codemirror--jsx`
3. Build Sandpack and try to consume it on another project: `yarn build` and `yarn dev:landing` to test.

---

- [x] Upgrade and deprecate CodeMirror dependencies;
- [x] Highlight `<foo>` differently from `<Foo>`; 

Closes #468
Potential fix for #666
Related to #668
